### PR TITLE
Fix dockerfile

### PR DESCRIPTION
**Pull request by orkunkl**
* Fix dependencies
* Set runtime os to latest(stable version)
* Remove user configurations to make things simpler, you can discard the change if you want.

### CODE REVIEW

**Review by colin-axner — APPROVED** [review]
LGTM, @marbar3778 could you take a quick peak :pray:

File: Dockerfile [hunk 1]
```diff
@@ -7,25 +7,17 @@ COPY . .
...
-    apk --no-cache add make=4.2.1-r2 git=2.24.3-r0 && \
+    apk --no-cache add make git && \
     make install
 
-FROM alpine:edge
+FROM alpine:latest
 
 ENV RELAYER /relayer
 
-RUN addgroup rlyuser && \
```
Comment (tac0turtle) [inline Dockerfile:17]: Looks fine, but why use the sudo user?

Comment (colin-axner) [reply Dockerfile:17]: I concur, could we add it back? Using users aligns with gaia dockerfile and anyone using a relayer in production should know how to run the relayer in a docker container as a regular user

**Comment by colin-axner** [comment]
#462 includes the relevant changes

**Pull request closed without merging**